web/src/Components/Combinations/CombinationCard.tsx

```diff
@@ -69,6 +86,20 @@ export const CombinationCard = ({
     } else {
       setTotalDensity(Math.round(newDensity * 10) / 10)
     }
+
+    CombinationAPI.postCombinationApi(apiToken, Object.values(combination.values))
+      .then(response => {
+        let newBridge: Bridge = { ...bridge, [combination.name]: response.data.bridge }
+        setBridge(newBridge)
+        let graphData: GraphData[] = findGraphData(sizeFractions, newBridge)
+        setD10(findDValue(graphData, 10, combination.name))
+        setD50(findDValue(graphData, 50, combination.name))
+        setD90(findDValue(graphData, 90, combination.name))
```

Comment:
Would it be better to set these values in the Combination object at the same time the graph is updated?
To avoid calculating the graph multiple times